packages/push-platform-notifications-controller/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "lib": [
+      "esnext",
+      "dom"
+    ],
+    "baseUrl": "./"
+  },
+  "references": [
+    {
+      "path": "../base-controller/tsconfig.build.json"
+    }
+  ],
+  "include": [
+    "../../../types",
+    "./src/encryption",
+    "./src/mocks",
```

Comment:
You don't have to be explicit about these — `./src` is a shortcut for `./src/**`:

``` suggestion
```